index.html

```diff
@@ -59,72 +59,21 @@ <h3>About the shelter “Cozy House”</h3>
   <section class="pets">
     
     <div class="container">
+      <div class="modal-content">
```

Comment:
Let's wrap modal-content in one more div with `position:relative;` it will help to arrange `icon-close` button